### DISCUSSION TITLE
[2-direct] 支持通配吗？

### DISCUSSION THREAD

**Discussion by loveinlastnight** [SeaHOH/GotoX#131] [问答/咨询]
[2-direct] #代理直连（兼容性较好）,如上传大文件失败,请换用 forward

这里的域名可以使用通配符吗？
比如有a.example.com; b.example.com; c.example.com;  .....
可用 *.example.com = 1.1.1.1这样来写。有时子域名多了，很麻烦。

**Comment by SeaHOH** [comment]
请自行查看配置文件顶部注释说明和默认规则示例。
